core/src/main/java/org/bitcoinj/core/Wallet.java

```diff
@@ -1856,6 +1895,65 @@ private void receive(Transaction tx, StoredBlock block, BlockChain.NewBlockType
         saveNow();
     }
 
+    /**
+     * Finds if tx is NOT spending other txns which are in the specified confidence type
```

Comment:
Again this can just be

// Finds if tx is NOT spending other txns which are in the specified confidence type